lib/configproxy.js

```diff
@@ -174,6 +175,7 @@ ConfigurableProxy.prototype.add_route = function (path, data) {
     this.routes[path] = data;
     this.trie.add(path, data);
     this.update_last_activity(path);
+    this.statsd.increment('route.add', 1);
```

Comment:
Nope - if you look at bin/configurable-http-proxy, I basically setup a stub statsd object that does nothing. This allows me to not need to guard against a null statsd everywhere.